js/gallery.js

```diff
@@ -12,9 +12,43 @@ window.onload = function chargeImages() {
     "./../assets/images/gallery/8.jpg"
   ];
   let div1 = document.getElementById("divImagesContainer").children;
-  for (let i = 0; i < div1.length; i++) {
-    div1[i].firstElementChild.style.backgroundImage = `url('${
-      imagesGallery[i]
-    }')`;
+  let imagesContainer = document.querySelector('#divImagesContainer');
+
+  for (let i = 0; i < imagesGallery.length; i++) {
+    imagesContainer.innerHTML += `
```

Comment:
please don't update the HTML each time you create an element, create a variable and update the HTML one only time.

Why? 
https://www.phpied.com/rendering-repaint-reflowrelayout-restyle/